clients/js/scripts/rename-cjs.js

```diff
@@ -0,0 +1,15 @@
+/** Renames `.js` files in the /lib/cjs folder to `.cjs` to appease ESM clients. */
+
+import { promises as fs } from 'node:fs';
+import url from 'node:url';
+
+process.chdir(url.fileURLToPath(import.meta.url + '/../../lib/cjs'));
+for (const filename of await fs.readdir('.')) {
+  if (!/\.js$/.test(filename)) continue;
+  const cjs = await fs.readFile(filename, 'utf-8');
+  await fs.writeFile(
+    filename.replace('.js', '.cjs'),
+    cjs.replaceAll(/require\("(.\/[\w_\-.]+)\.js"\)/g, 'require("$1.cjs")'),
```

Comment:
eeeeeeeew

i do not approve

Reply:
rly? it gets the job done

Reply:
`\w` includes `_`

Reply:
Report upstream, that's a bug in regex :rofl:

Reply:
nuh uh